server/events.go

```diff
@@ -2105,6 +2106,7 @@ func (a *Account) statz() *AccountStat {
 	leafConns := a.numLocalLeafNodes()
 	return &AccountStat{
 		Account:    a.Name,
+		Name:       a.LogicalName,
```

Comment:
Any objection to adding this to a few other places? connz mainly

Reply:
Yes, will add to the rest of the endpoints.